subsys/bluetooth/mesh/light_hue_srv.c

```diff
@@ -131,7 +131,7 @@ static void hue_set(struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
 	(void)bt_mesh_lvl_srv_pub(&srv->lvl, NULL, &lvl_status);
 
 	if (IS_ENABLED(CONFIG_BT_MESH_SCENE_SRV)) {
-		bt_mesh_scene_invalidate(&srv->lvl.scene);
+		bt_mesh_scene_invalidate(srv->model);
```

Comment:
Hm, yes. There's no reasonable way to fix this without doing NCSDK-9878, and this currently works, as the model pointer is only used to determine the element index. Once that bugfix task is done, this will be the right solution, so I suggest we just leave it.